extensions/adobe/experience/offer-management/content/image-content.schema.json

```diff
@@ -0,0 +1,68 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/content/image-content",
+  "meta:extends": ["https://ns.adobe.com/experience/offer-management/representation-content"],
+  "meta:abstract": false,
+  "meta:extensible": true,
+  "type": "object",
+  "title": "Image Content",
+  "description": "Image content used in an offer representation",
+  "definitions": {
+    "image-content": {
+      "type": "object",
+      "title": "Image Content",
```

Comment:
how can you have "image content" in a string in JSON?  That doesn't make any sense.

Reply:
These are links to images. See properties below such as xdm:imageUrl